.devcontainer/Dockerfile

```diff
@@ -1,21 +1,13 @@
-FROM mcr.microsoft.com/mirror/docker/library/ubuntu:22.04
+FROM mcr.microsoft.com/devcontainers/typescript-node:20-bookworm
 
-RUN apt-get -y update &&  apt upgrade -y && apt install curl -y
-RUN apt-get -y install git
+WORKDIR /workspaces/azure-rest-api-specs
 
-# install node and npm
-RUN curl -sL https://deb.nodesource.com/setup_16.x | bash -
-RUN apt -y install nodejs
-RUN npm install -g npm@9.5.0
+RUN wget -q https://packages.microsoft.com/config/ubuntu/22.04/packages-microsoft-prod.deb \
```

Comment:
Do you also know what this one does?

Reply:
yes, it configures the microsoft packages sources:

https://learn.microsoft.com/en-us/linux/packages#how-to-install-microsoft-software-packages-using-the-linux-repository

Reply:
I think we should be able to simplify the Dockerfile even further.  It should just need:

1. Node 20
2. clone repo
3. npm ci